Doc/library/functions.rst

```diff
@@ -965,7 +965,7 @@ are always available.  They are listed here in alphabetical order.
       *classinfo* can be a :ref:`types-union`.
 
 
-.. function:: issubclass(class, classinfo)
+.. function:: issubclass(cls, class_or_tuple , /)
```

Comment:
I'd leave the param name untouched and instead consider updating the docstring.

```suggestion
.. function:: issubclass(class, classinfo , /)
```

Reply:
There's also a typo (stray space):

```suggestion
.. function:: issubclass(cls, class_or_tuple, /)
```

or

```suggestion
.. function:: issubclass(cls, classinfo, /)
```